@types/commercetools-frontend__ui-kit/index.d.ts

```diff
@@ -0,0 +1,16 @@
+declare module '@commercetools-frontend/ui-kit' {
```

Comment:
We really need to expose types from ui-kit!

I'll try to work on [this approach](https://github.com/commercetools/ui-kit/pull/860) until we decide to convert the components to TS as well. This should buy us some time.

/cc @montezume